src/constants.rs

```diff
@@ -0,0 +1,26 @@
+pub const FACES_SIZE: usize = 14;
+pub const FACES: [&[u8]; FACES_SIZE] = [
```

Comment:
I'd keep the formatting consistent here with `ACTIONS`

Another optimisation we might be able to add is appending the spaces within the strings themselves here so we don't have to specifically `out.write_all(b" ")` later on?

Reply:
They are in the same format. I don't know what your talking about

Reply:
Ah sorry I just meant how `ACTIONS` are on new lines whereas the `FACES` are on the same line

Reply:
Oh, sorry about that. That's a product of using `rustfmt` to format my code.

Reply:
I changed the formatting to be consistent

Reply:
I was using `rustfmt` earlier and it seemed to be quite inconsistent :/

Reply:
Yeah, `rustfmt` prioritizes readabiility over consistency.